README.md

```diff
@@ -3,37 +3,97 @@
 This sample app demonstrates how to use a function, workflow, and trigger to
 send a greeting to channel.
 
+**Guide Outline**:
+
+- [Setup](#setup)
+  - [Install the Slack CLI](#install-the-slack-cli)
+  - [Clone the Template](#clone-the-template)
```

Comment:
```suggestion
  - [Clone the Sample](#clone-the-sample)
```

Reply:
Eagle eyes!

Reply:
Should the heading on line 33 be changed to "Clone the Template" for the link's linking needs?